test/performance/rest-interface/config/tripod-config.json

```diff
@@ -646,6 +646,50 @@
                             "predicates": ["rdf:type"]
                         }
                     ]
+                },
+                {
+                    "_id": "t_conditional_creators",
+                    "type": ["bibo:Book", "bibo:Document"],
+                    "from": "CBD_testing",
+                    "counts": [
+                        {
+                            "fieldName": "!creatorCount",
```

Comment:
Is there a reason why the field names have a ```!``` in front of them? It looks a bit confusing to me - "not the creatorCount"?